pkg/scheduler/eventhandlers.go

```diff
@@ -461,6 +461,9 @@ func nodeSchedulingPropertiesChange(newNode *v1.Node, oldNode *v1.Node) *framewo
 	if nodeConditionsChanged(newNode, oldNode) {
 		return &queue.NodeConditionChange
 	}
+	if nodeAnnotationsChanged(newNode, oldNode) {
```

Comment:
Is there no test for `nodeSchedulingPropertiesChange` function? If so, could you create the one instead of creating the test for `nodeAnnotationsChanged`?

Reply:
Currently, there is no unit test for the nodeSchedulingPropertiesChange function. In the previous code, there were separate unit tests for each event. Are you suggesting that I consolidate them into a single unit test?

Reply:
Ah, it's 🙆‍♂️ then

Reply:
Sorry I was unclear. 
I meant you can just keep the existing unit tests and one new test function for nodeAnnotationsChanged is ok. 🙏

Reply:
I consolidated all the test cases for node change events into the `TestNodeSchedulingPropertiesChange` function.
Now, if we need to add additional node event cases, we can simply add them as test cases within the existing `TestNodeSchedulingPropertiesChange` function, rather than creating new test functions.

Reply:
+1 for just add a `TestNodeAnnotationsChanged` test, that seems more clear :)

Reply:
Actually, I would prefer we have a test for `nodeSchedulingPropertiesChange` only, as this is the higher level method that directly interacts with the event handlers.

Testing at a higher level reduces the risk of bugs where multiple of these tiny functions interact.